ingestion/functions/common/parsing_lib.py

```diff
@@ -183,7 +181,7 @@ def write_to_server(
                 # We can consider separating geocoding issues, but for now classifying it
                 # as a validation problem is pretty reasonable.
                 # The motivation for continuing past 207 errors is https://github.com/globaldothealth/list/issues/1849
```

Comment:
remove this comment?

Reply:
Useful for historians looking at the code, so keeping it for now :)